addons/binding/org.openhab.binding.evohome/src/main/java/org/openhab/binding/evohome/internal/api/EvohomeApiClientV2.java

```diff
@@ -215,4 +218,35 @@ public ControlSystem getControlSystem(int id) {
         return null;
     }
 
+    /**
+     * Returns the specified Heating Zone or null if one can't be found
+     * @return
+     */
+    @Override
+    public ZoneStatus getHeatingZone(int locationId, int zoneId) {
```

Comment:
I'd like to follow your lead and leave the == null checks out and just let it throw... Agree?